src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,375 @@
+
+Astro [provides built-in functions](#querying-content-collections) for querying your content data files by collection directory name. This means every content file in `src/content/` **must** belong to a collection directory.
+
+With this organization, you can retrieve a set of entries based on the collection name, and optionally enforce frontmatter types with a schema.
+
+To create a new collection, add a new directory to `src/content/`, and add Markdown or MDX entries that share frontmatter properties. The following example shows two collections: `blog` and `newsletter`. 
+
+```bash "newsletter/" "blog/"
+src/content/
+│   # All blog posts have the same frontmatter properties
+├── blog/
+│   ├── columbia.md
+│   ├── endeavour.md
+│   └── enterprise.md
+│   # All newsletters have the same frontmatter properties
+└── newsletter/
+    ├── week-1.md
+    ├── week-2.md
+    └── week-3.md
+```
+
+### Organizing with nested directories
+
+Collections are **top-level folders** within `src/content`. You cannot nest collections, but you may use nested directories within a collection to better organize a collection's content. All nested directories will share the same schema defined for the top-level collection.
+For example, you can use this structure for internationalization:
+
+```bash
+src/content/
+└── docs/
+    │   # docs schema applies to all nested directories 👇
+    ├── en/
+    ├── es/
+    └── ...
+```
+
+All nested directories will share the same schema defined for the top-level collection, if any (**docs** in this example).
+
+## Defining a collection schema
+
+Schemas are an optional way to enforce frontmatter types in a collection. Astro uses [Zod](https://github.com/colinhacks/zod) to validate your frontmatter with schemas as [Zod objects](https://github.com/colinhacks/zod#objects).
+
+To configure schemas, create a `src/content/config.{js|mjs|ts}` file. This file should:
+
+1. Import the `defineCollection` and `z` utilities from `astro:content`. These are used to define a `schema` for each collection.
+2. Export a `collections` object, with each object key corresponding to the collection's folder name.
+
+For example, say you have a `src/content/engineering-blog` collection, where every entry should have a `title`, list of `tags`, and an optional `image` URL. You can specify each expected property like so:
+
+```ts
+// src/content/config.ts
+import { z, defineCollection } from 'astro:content';
+
+const engineeringBlog = defineCollection({
+  schema: {
+    title: z.string(),
+    image: z.string().optional(),
+    tags: z.array(z.string()),
+  },
+});
+
+export const collections = {
+  // Don't forget 'quotes' for collection names containing dashes
+  'engineering-blog': engineeringBlog,
+};
+```
+
+### Schema data types with Zod
+
+Markdown and MDX frontmatter can contain booleans, strings, numbers, objects, and arrays. Each frontmatter property must be listed in your schema object along with its data type.
```

Comment:
Zod has only been mentioned in the headline. So at this point, a reader who doesn't know Zod may be confused what Zod actually is. A little introduction would help. Something like:

```suggestion
Markdown and MDX frontmatter can contain booleans, strings, numbers, objects, and arrays. Each frontmatter property must be listed in your schema object along with its data type. To define and validate this schema, we use a library called [zod](https://github.com/colinhacks/zod), which we use via the `z` import.
```